### PR TITLE
Coll changes to compile on babashka

### PR DESCRIPTION
**Pull request by wilkerlucio**

### CODE REVIEW

File: src/main/com/wsscode/misc/coll.cljc [hunk 1]
```diff
@@ -1,7 +1,11 @@
 (ns com.wsscode.misc.coll
   (:require
     [clojure.set :as set])
-  #?(:clj
+  #?(:bb
```
Comment (borkdude) [inline src/main/com/wsscode/misc/coll.cljc:4]: @wilkerlucio Why is this needed? Same as `:clj` ?

File: src/main/com/wsscode/misc/coll.cljc [hunk 2]
```diff
@@ -234,7 +238,11 @@
      m)))
 
 (defn native-map? [x]
-  #?(:clj  (or (instance? clojure.lang.PersistentArrayMap x)
+  #?(:bb   (let [cn (.getName (class x))]
+             (contains? {"clojure.lang.PersistentArrayMap"
```
Comment (borkdude) [inline src/main/com/wsscode/misc/coll.cljc]: These should be fixed on bb master.

File: src/main/com/wsscode/misc/coll.cljc (hunk 1 above)
Comment (wilkerlucio) [reply src/main/com/wsscode/misc/coll.cljc:4]: Its not, if you expand the CLJ version, it imports more classes, that fail to import in Babashka

File: src/main/com/wsscode/misc/coll.cljc (hunk 2 above)
Comment (wilkerlucio) [reply src/main/com/wsscode/misc/coll.cljc]: Cool, removed

File: src/main/com/wsscode/misc/coll.cljc (hunk 1 above)
Comment (wilkerlucio) [reply src/main/com/wsscode/misc/coll.cljc:4]: The `clojure.lang.RT` and `java.util.Iterator` classes
